graphene_django_extras/mutation.py

```diff
@@ -210,7 +210,7 @@ def create(cls, root, info, **kwargs):
         if non_required_fields:
             non_required_fields_list = [field.name for field in non_required_fields]
             m2m_fields = [
-                field.attname for field in cls._meta.model._meta.local_many_to_many if field.attname not in non_required_fields_list]
+                field.attname for field in cls._meta.model._meta.local_many_to_many if field.attname in non_required_fields_list]
```

Comment:
What is this fix?

Reply:
Seperately adding m2m fields which are non_required as some of m2m are read only in non req which serializer ignores to add or update.